src/components/Contact.tsx

```diff
@@ -0,0 +1,89 @@
+import { BiBell, BiMusic } from 'react-icons/bi'
+
+import { Avatar, Flex, Heading, Text, Icon } from '@chakra-ui/react'
+
+import Status from './Status'
+
+export type ContactProps = {
+  name: string
+  src: string
+  statusText?: string
+  song?: string
+  notification?: number
+  alert?: boolean
+  status?: string | undefined
```

Comment:
```suggestion
  status?: string
```

quando voce passa o optinal `?` nao precisa passar o undefined.
O Typescript ja entende que quando voce passa o optional, a chave pode ser `undefined`